### PR TITLE
[2/2] Translations overhaul

### PR DESCRIPTION
**Pull request by tomquirk**
Let's switch it up.

This PR creates a `messages.pot` file. `.pot` is considered a "template" file for our strings.

We send this file to crowdin. Translators translate. Then we periodically (weekly) download the translations (`messages.po` files) and merge them into `main`.

Note that `en` has been removed. This was previously our source file, but now `messages.pot` has assumed that role.

**What does this mean for devs?**

Devs still run `yarn i18n:extract` for new strings. **But, PRs will only ever contain one file: `messages.pot` 🎉** . PRs should _never_ contain changes to `messages.po` files.

**What about Crowdin?**

I've made all the changes on the Crowdin side. Translators will use it as normal, and all the previous translation approvals remain in-tact.

**Bugs?**

I'm fairly confident we'll be fine, but given the unforeseen complications we've had with internationalization in the past, I wouldn't be surprised to see bugs. Let's monitor the next few days carefully.

There may also be weird kinks to work through in the CI and linting. I'll ping the team to make everyone aware.

### CODE REVIEW

File: src/components/PayWarningModal.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ export default function PayWarningModal({
       </h2>
       <p style={{ fontWeight: 500 }}>
         <Trans>
-          The
+          The{' '}
```
Comment (tomquirk) [inline src/components/PayWarningModal.tsx:28]: bug, soz

File: scripts/lint-translations.sh [hunk 2]
```diff
@@ -1,25 +1,13 @@
-# extract new strings
-yarn i18n:extract
-
 # get all translation files that were updated
 # from `yarn i18n:extract`
-LOCALE_CHANGED_FILES=$(git diff --name-only src/locales)
+CHANGED_FILES=$(git diff --name-only src/locales/**/*.po)
```
Comment (tomquirk) [inline scripts/lint-translations.sh:3]: PRs shouldn't contain any changes to `.po` files anymore. Should only contain changes to `.pot` file.

We'll run this script in the CI for PRs :)

File: .github/workflows/crowdin-download.yml [hunk 3]
```diff
@@ -26,5 +26,5 @@ jobs:
           create_pull_request: true
           project_id: '492549'
           token: ${{ secrets.CROWDIN_PERSONAL_TOKEN }}
-          source: 'src/locales/en/messages.po'
-          translation: 'src/locales/%two_letters_code%/%original_file_name%'
+          source: 'src/locales/messages.pot'
```
Comment (tomquirk) [inline .github/workflows/crowdin-download.yml:29]: source now the `.pot`

File: .github/workflows/crowdin-download.yml [hunk 4]
```diff
@@ -26,5 +26,5 @@ jobs:
...
+          translation: 'src/locales/%two_letters_code%/messages.po'
```
Comment (tomquirk) [inline .github/workflows/crowdin-download.yml:30]: `%original_file_name%` comes back as `messages.pot`, but we want `messages.po` - this ensures that.

File: scripts/lint-translations-template.sh [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# extract new strings
```
Comment (tomquirk) [inline scripts/lint-translations-template.sh:1]: In this script, we check if the template .pot file is valid and up to date.

File: .linguirc.json [hunk 6]
```diff
@@ -11,9 +11,5 @@
     "origins": false
   },
   "orderBy": "messageId",
-  "fallbackLocales": {
-    "default": "en"
-  },
-  "locales": ["en", "zh", "ru", "tr", "es", "pt", "fr"],
-  "sourceLocale": "en"
+  "locales": ["zh", "ru", "tr", "es", "pt", "fr"]
```
Comment (tomquirk) [inline .linguirc.json:14]: Remove english as a lingui lang. The source file is now the `.pot`.

If a translation for a string doesn't exist for a given lang, lingui falls back to the message id.(which is the english string)

File: src/components/PayWarningModal.tsx (hunk 1 above)
Comment (tomquirk) [inline src/components/PayWarningModal.tsx:28]: passing fix, my bad

**Review by johnnyd-eth — APPROVED** [review]
Awesome LGTM let's see how she goes!

**Pull request merged**